src/main/java/org/gridsuite/directory/server/DirectoryController.java

```diff
@@ -263,4 +263,13 @@ public ResponseEntity<Page<DirectoryElementInfos>> searchElements(
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON)
                 .body(service.searchElements(userInput, directoryUuid));
     }
+
+    @GetMapping(value = "/directories/uuid", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
this endpoint doesn't consume JSON_VALUE

Reply:
:+1:

Reply:
Done